app/javascript/spotlight/admin/croppable_modal.js

```diff
@@ -0,0 +1,106 @@
+import Crop from 'spotlight/admin/crop';
+
+export default class CroppableModal {
+
+  attachModalHandlers() {
+    // Attach handler for when modal first loads, to show the cropper
+    this.attachModalLoadBehavior();
+    // Attach handler for save by checking if clicking in the modal is on a save button
+    this.attachModalSaveHandler();
+  }
+
+  attachModalLoadBehavior() {
+    var context = this;
+    // Listen for event thrown when modal is displayed with content
+    document.addEventListener('show.blacklight.blacklight-modal', function(e) {      
+      var dataCropperDiv = $('#blacklight-modal [data-behavior="iiif-cropper"]');
+      
+      if(dataCropperDiv) {
+        var dataCropperKey = dataCropperDiv.data('cropper-key');
+        var itemIndex = dataCropperDiv.data('index-id');
+        //var iiifFields = context.getIIIFObject(dataCropperKey, itemIndex);
+        // The region field is set separately within the modal div
+        //iiifFields['iiifRegionField'] = context.setRegionField(dataCropperKey, itemIndex);
+        new Crop(dataCropperDiv, false).render();
+        //context.attachModalSaveHandler(dataCropperKey);
```

Comment:
I think you might have meant to delete or uncomment some of these?

Reply:
yes